automated install/basic-install.sh

```diff
@@ -1008,27 +1008,27 @@ remove_old_pihole_lighttpd_configs() {
     local confenabled="/etc/lighttpd/conf-enabled/15-pihole-admin.conf"
 
 
-    if [[ -d "/etc/lighttpd/conf.d" ]]; then
-        sed -i '/include "\/etc\/lighttpd\/conf.d\/pihole-admin.conf"/d' "${lighttpdConfig}"
-
-        if [[ -f "${condfd}" ]]; then
-            rm "${condfd}"
+    if [[ -f "${lighttpdConfig}" ]]; then
+        if grep -q -F 'include "/etc/lighttpd/conf.d/pihole-admin.conf"' "${lighttpdConfig}"; then
```

Comment:
Why did you add the `if` again?

If the file exists _(line 1011 makes sure it does)_ there's no need to test the existence of the line with `grep`. Just use `sed`.